private/pkg/app/appcmd/appcmd.go

```diff
@@ -167,6 +170,40 @@ func run(
 	// commands.
 	cobraCommand.CompletionOptions.DisableDefaultCmd = true
 
+	// Add an explicit "completion" command to the root to cover the most
+	// common use case.
+	if command.IsRoot {
```

Comment:
This should be in the same `if` statement as the existing `bash-completion` etc commands.

Reply:
Done